packages/pool/test/StateUtils.sol.js

```diff
@@ -566,84 +568,36 @@ describe("setUnstakeWaitPeriod", function () {
   });
 });
 
-describe("setAprUpdateCoefficient", function () {
+describe("setAprUpdateStep", function () {
   context("Caller is DAO Agent", function () {
-    context(
-      "APR update coefficient to be set is larger than 0 and smaller than or equal to 1,000,000,000",
-      function () {
-        it("sets APR update coefficient", async function () {
-          await api3Pool
-            .connect(roles.randomPerson)
-            .setDaoApps(
-              roles.agentAppPrimary.address,
-              roles.agentAppSecondary.address,
-              roles.votingAppPrimary.address,
-              roles.votingAppSecondary.address
-            );
-          const oldAprUpdateCoefficient = await api3Pool.aprUpdateCoefficient();
-          const newAprUpdateCoefficient = ethers.BigNumber.from(
-            "50" + "000" + "000"
-          );
-          await expect(
-            api3Pool
-              .connect(roles.agentAppPrimary)
-              .setAprUpdateCoefficient(newAprUpdateCoefficient)
-          )
-            .to.emit(api3Pool, "SetAprUpdateCoefficient")
-            .withArgs(oldAprUpdateCoefficient, newAprUpdateCoefficient);
-          expect(await api3Pool.aprUpdateCoefficient()).to.equal(
-            newAprUpdateCoefficient
-          );
-          await expect(
-            api3Pool
-              .connect(roles.agentAppSecondary)
-              .setAprUpdateCoefficient(newAprUpdateCoefficient)
-          )
-            .to.emit(api3Pool, "SetAprUpdateCoefficient")
-            .withArgs(newAprUpdateCoefficient, newAprUpdateCoefficient);
-          expect(await api3Pool.aprUpdateCoefficient()).to.equal(
-            newAprUpdateCoefficient
-          );
-        });
-      }
-    );
-    context(
-      "APR update coefficient to be set is 0 or larger than 1,000,000,000",
-      function () {
-        it("reverts", async function () {
-          await api3Pool
-            .connect(roles.randomPerson)
-            .setDaoApps(
-              roles.agentAppPrimary.address,
-              roles.agentAppSecondary.address,
-              roles.votingAppPrimary.address,
-              roles.votingAppSecondary.address
-            );
-          const newAprUpdateCoefficient1 = ethers.BigNumber.from(0);
-          await expect(
-            api3Pool
-              .connect(roles.agentAppSecondary)
-              .setAprUpdateCoefficient(newAprUpdateCoefficient1)
-          ).to.be.revertedWith("Invalid value");
-          const newAprUpdateCoefficient2 = ethers.BigNumber.from(
-            "50" + "000" + "000" + "000"
-          );
-          await expect(
-            api3Pool
-              .connect(roles.agentAppSecondary)
-              .setAprUpdateCoefficient(newAprUpdateCoefficient2)
-          ).to.be.revertedWith("Invalid value");
-        });
-      }
-    );
+    it("sets APR update coefficient", async function () {
```

Comment:
`aprUpdateStep` can be set to 100%+
https://github.com/api3dao/api3-dao/blob/5ecf1b09b672a16ffc6ba1969dd3929e7865d4a5/packages/pool/contracts/StateUtils.sol#L342
That's because we may want to increase APR by 250% steps too for example (we wouldn't but there is no theoretical reason why we shouldn't be able to)